driver/modern_bpf/programs/tail_called/events/syscall_dispatched_events/accept.bpf.c

```diff
@@ -77,7 +77,7 @@ int BPF_PROG(accept_x,
 		auxmap__store_socktuple_param(auxmap, (int32_t)ret, INBOUND, NULL);
 
 		/* Collect parameters at the beginning to  manage socketcalls */
-		unsigned long args[1];
+		unsigned long args[1] = {0};
 		extract__network_args(args, 1, regs);
```

Comment:
I'm being dense here, but couldn't we use an unsigned long directly here?
```c
unsigned long args = 0;
extract__network_args(&args, 1, regs);
```
We could even simplify it to be `sockfd` directly which is used on the next line.

Reply:
Done, thank you!